apps/src/p5lab/utils.js

```diff
@@ -17,3 +17,15 @@ export function arrayEquals(a, b) {
 
   return true;
 }
+
+// Creates and returns a helper object that represents the outer bounds of a sprite.
+// Used for comparing the positions of sprites. For example, checking whether
+// one sprite is "standing" on top of another.
```

Comment:
nit: mind adding an explainer of what the numbers mean? This might be me not knowing how p5 works, are these pixel locations or some other coordinate system?

Reply:
Just pushed a comment to explain!